range_sensor_layer/include/range_sensor_layer/range_sensor_layer.h

```diff
@@ -79,6 +82,10 @@ class RangeSensorLayer : public costmap_2d::CostmapLayer
   std::vector<ros::Subscriber> range_subs_;
   double min_x_, min_y_, max_x_, max_y_;
 
+  bool use_decay_;
+  double pixel_decay_;
+  double US_transform_waiting_time_;
```

Comment:
What is US here?

Reply:
Ultra Sound, I used US to make a shorter variable name

Reply:
Since the sensor isn't assumed to be an ultrasound, can we just make it the more generic `transform_tolerance_`? Standard is not to use capital letters anyway.